src/pages/LoginPage.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC } from "react";
+import { NavLink, useNavigate } from "react-router-dom";
+import { Button, Form, Container, Card } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as Yup from "yup";
+import ky from "ky";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import "../styles/Auth.scss";
+import { useDispatch } from "react-redux";
+import { loginUser } from "../store/actionCreators";
+
+type UserLoginForm = {
+  email: string;
+  password: string;
+};
+
+type ResponseLogin = {
+  token: string;
+  message: string;
+};
+
+const LoginPage: FC = () => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const validationSchemaLogin = Yup.object().shape({
+    email: Yup.string().required("Email обязателен").email("Неверный email"),
+    password: Yup.string()
+      .required("Пароль обязателен")
+      .min(6, "Пароль должен состьять из 6 символов")
+      .max(20, "Пароль не может иметь больше 20 сиволов"),
+  });
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<UserLoginForm>({
+    resolver: yupResolver(validationSchemaLogin),
+  });
+  const onSubmitRegistration: any = async (data: UserLoginForm) => {
+    try {
+      const response: ResponseLogin = await ky
```

Comment:
Let's create the HTTP request wrapper there are will be error handler and the API routers collection